Packages/com.unity.inputsystem/InputSystem/Utilities/ArrayHelpers.cs

```diff
@@ -147,6 +147,25 @@ public static bool HaveEqualElements<TValue>(TValue[] first, TValue[] second, in
             return true;
         }
 
+        public static bool EqualSets<TValue>(TValue[] first, TValue[] second)
```

Comment:
If we do keep this method (see other comment above about replacing it with All) then the comment in the body should be promoted to an xml comment

Reply:
I removed the method.